python/mkdocs/docs/index.md

```diff
@@ -529,10 +529,10 @@ The goal is that the python library will provide a functional, performant subset
 |:------------------------|:-----:|:------:|
 | Get Schema              |    X  |   X    |
 | Get Snapshots           |    X  |   X    |
-| Plan Scan               |    X  |        |
-| Plan Scan for Snapshot  |    X  |        |
+| Plan Scan               |    X  |   X    |
+| Plan Scan for Snapshot  |    X  |   X    |
 | Update Current Snapshot |    X  |        |
-| Set Table Properties    |    X  |   X    |
+| Set Table Properties    |    X  |        |
```

Comment:
You might want to include a line for `Set Namespace Properties` because that is supported.